admin/cincoctrl/config/settings/production.py

```diff
@@ -1,26 +1,40 @@
 # ruff: noqa: E501
-import logging
+# import logging
 
-import sentry_sdk
-from sentry_sdk.integrations.django import DjangoIntegration
-from sentry_sdk.integrations.logging import LoggingIntegration
-from sentry_sdk.integrations.redis import RedisIntegration
+# import sentry_sdk
+# from sentry_sdk.integrations.django import DjangoIntegration
+# from sentry_sdk.integrations.logging import LoggingIntegration
 
 from .base import *  # noqa: F403
 from .base import DATABASES
 from .base import INSTALLED_APPS
 from .base import env
 
+DEBUG = True
```

Comment:
I don't think we want this as always `True` in the production settings. Is this just for development?

Reply:
Yeah, I conflated "deployed" with "production" - I think some follow up work involves creating a "stage.py" settings file and related resources. I think these three commits include things we'll want to roll back for production: 

https://github.com/ucldc/cinco/commit/4ecb929d1a6215b057b377554979db51ecb8f02f
https://github.com/ucldc/cinco/commit/580f08c86142f132381b2e6faff3a8753d0a4c24
https://github.com/ucldc/cinco/commit/0530838c9f3514b1928a7de4b96c1176235d66c1